sumpy/symbolic.py

```diff
@@ -163,13 +163,13 @@ def UnevaluatedExpr(x):  # noqa: N802
     def doit(expr: Expr) -> Expr:
         return expr
 
-    def unevaluated_pow(a: Expr, b: complex | Expr) -> ArithmeticExpr:
```

Comment:
I don't think `ArithmeticExpr` is used anywhere else, so it can probably go? I just added it for this because sympy is nice enough to evaluate `Pow(2, 3)`, but that's not actually the case here..